conda_rpms/config.yaml.example

```diff
@@ -3,3 +3,9 @@ rpm:
 
 install:
     prefix: '/opt/scitools'
+
+taggedenv:
+    summary: 'The {{ name }}-{{ tag }} environment.'
```

Comment:
I don't think this should be configured in this way. In general, it should be on a per environment basis.
That said, this would make an excellent *default*.